x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field_helpers.tsx

```diff
@@ -91,14 +91,16 @@ export const RenderRuleName: React.FC<RenderRuleNameProps> = ({
         return;
       }
 
-      openRightPanel({
-        id: RulePanelKey,
-        params: {
-          ruleId,
+      openFlyout({
```

Comment:
I was actually wondering while reviewing the changes made to the `kbn-expandable-flyout` `reducers.ts` file if we should also add to the history for the `openRightPanel` for that very reason that developers might not be using the correct api...
I guess this is a downside of having a flexible api where `openFlyout({right:{...}})` does the same thing as `openRightPanel{...}).

But then I realized that we use the `openRightPanel` when changing tabs so we can't do that 😆 

Good catch changing this one here!